external.desc

```diff
@@ -0,0 +1 @@
+name: CUSTOM_RADXA_ZEROW
```

Comment:
Change this to CUSTOM_AUDIO_INJECTOR

Reply:
I expect you';ll have to change :
BR2_EXTERNAL_CUSTOM_RADXA_ZEROW_PATH
To :
BR2_EXTERNAL_CUSTOM_AUDIO_INJECTOR_PATH